index.html

```diff
@@ -10,6 +10,8 @@
 
   <body>
     <h1>That's the theme</h1>
```

Comment:
The changes added here seem to be for testing purposes. It's recommended to update the content to something more meaningful for the project.